components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -259,7 +432,20 @@ service Policies {
     option (chef.automate.api.policy).action = "read";
     option (chef.automate.api.iam.policy).resource = "iam:projects";
     option (chef.automate.api.iam.policy).action = "iam:projects:list";
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "projects";
+		};
   };
+  /*
+  Delete a project
+
+  Deletes a project. Cascade deletes the project from any resources tagged with it.
+  If a resulting resource no longer belongs to any project, it goes into the (unassigned) project.
+
+  Also cascade deletes this project from any project list in all statements.
+  If the resulting project list for a given statement is empty, it is deleted.
+  If the resulting policy has no statements, it is also deleted.
```

Comment:
Similar to above, suggest remove "cascade delete".
Per @susanev's suggestion to remove "(unassigned)", I think deleting the second sentence would be fine.